Updated and fixed opentelemetry input plugin options configuration table. Fixes #2152. (#2153)
fluent/fluent-bit-docs: pipeline/inputs/opentelemetry.md

| `encode_profiles_as_log`            | Encode profiles received as text and ingest them in the logging pipeline.                                                                                                                     | `true`      |
| `host`                              | The hostname.                                                                                                                                                                                 | `localhost` |
| `http2`                             |                                                                                                                                                                                               | `true`      |
| `listen`                            | The network address to listen on.                                                                                                                                                             | `0.0.0.0`   |

> | `log_level`                         | Specifies the log level for output plugin. If not set here, plugin uses global log level in `service` section.                                                                                | `info`      |

Reviewer: ```suggestion
| `log_level`                         | Specifies the log level for this plugin. If not set here, the plugin uses the global log level specified in the `service` section of your configuration file.                                                                                | `info`      |
```
(I assume it should be for _this_ plugin, which isn't an output plugin, but if my suggestion is incorrect feel free to ignore :P)

> | `log_supress_interval`              | Suppresses log messages from output plugin that appear similar within a specified time interval. `0` no suppression.                                                                          | `0`         |

Reviewer: ```suggestion
| `log_supress_interval`              | Suppresses log messages from this plugin that appear similar within a specified time interval. `0` no suppression.                                                                          | `0`         |
```
(same as above)

| `net.accept_timeout`                | Set maximum time allowed to establish an incoming connection, this time includes the TLS handshake.                                                                                           | `10s`       |
| `net.accept_timeout_log_error`      | On client accept timeout, specify if it should log an error. When disabled, the timeout is logged as a debug message.                                                                         | `true`      |
| `net.backlog`                       | Set the backlog size for listening sockets.                                                                                                                                                   | `128`       |
| `net.io_timeout`                    | Set maximum time a connection can stay idle.                                                                                                                                                  | `0s`        |

> | `net.keepalive`                     | Enable or disable Keepalive support.                                                                                                                                                          | `true`      |

Reviewer: ```suggestion
| `net.keepalive`                     | Enable or disable keepalive support.                                                                                                                                                          | `true`      |
```
